pallets/auctions/src/weights.rs

```diff
@@ -0,0 +1,121 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Autogenerated weights for pallet_auction
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 3.0.0
+//! DATE: 2021-04-30, STEPS: [50, ], REPEAT: 20, LOW RANGE: [], HIGH RANGE: []
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 128
+
+// Executed Command:
+// target/release/subauction
+// benchmark
+// --chain=dev
+// --steps=50
+// --repeat=20
+// --pallet=pallet_auction
+// --extrinsic=*
+// --execution=wasm
+// --wasm-execution=compiled
+// --heap-pages=4096
+// --output=auction_weights.rs
+// --template=.maintain/pallet-weight-template.hbs
+
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+#![allow(clippy::unnecessary_cast)]
+
+use frame_support::{
+	traits::Get,
+	weights::{constants::RocksDbWeight, Weight},
+};
+use sp_std::marker::PhantomData;
+
+/// Weight functions needed for pallet_auction.
+pub trait WeightInfo {
+	fn create_auction() -> Weight;
+	fn update_auction() -> Weight;
+	fn bid() -> Weight;
+	fn destroy_auction() -> Weight;
+	fn close_auction() -> Weight;
+	fn claim() -> Weight;
+}
+
+/// Weights for pallet_auction using the subauction node and recommended hardware.
+pub struct BasiliskWeight<T>(PhantomData<T>);
+
+impl<T: frame_system::Config> WeightInfo for BasiliskWeight<T> {
+	fn create_auction() -> Weight {
+		(74_809_000 as Weight)
```

Comment:
How did you determine this weight and why is it so different from the one in the basilisk runtime?

Reply:
weights are dummies, i.e. random nrs. 
pending benchmarks but curious to first hear your thoughts on general interface design

Reply:
Please don't deploy with random values. At the minimum do a conservative educated guess that you expect to be hard to exploit. (e.g. pallet_xcm has had conservative weights for [some of its extrinsics](https://github.com/paritytech/polkadot/blob/27b839e5d4ffe2c13f19584052d3b3ae8910906a/xcm/pallet-xcm/src/lib.rs#L564-L565) for a while)